README.md

```diff
@@ -197,11 +199,14 @@ See [Bamboo.EmailPreviewPlug](https://hexdocs.pm/bamboo/Bamboo.EmailPreviewPlug.
 
 ## Mandrill Specific Functionality (tags, merge vars, etc.)
 
-Mandrill offers extra features on top of regular SMTP email like tagging, merge vars, and scheduling emails to send in the future. See [Bamboo.MandrillHelper](https://hexdocs.pm/bamboo/Bamboo.MandrillHelper.html).
+Mandrill offers extra features on top of regular SMTP email like tagging, merge
+vars, and scheduling emails to send in the future. See
+[Bamboo.MandrillHelper](https://hexdocs.pm/bamboo/Bamboo.MandrillHelper.html).
 
 ## Testing
 
-You can use the Bamboo.TestAdapter along with [Bamboo.Test] to make testing your emails a piece of cake.
+You can use the Bamboo.TestAdapter along with [Bamboo.Test] to make testing your
+emails a piece of cake.
```

Comment:
Minor: what do you think about using `easy` here instead of `a piece of cake`. I don't think the reference makes any more distinction and I could imagine a lot of non-english natives not understanding that temr.